indico/modules/events/registration/templates/display/event_header.html

```diff
@@ -52,11 +52,16 @@
     <tr>
         <td class="leftCol">{% trans %}Participants{% endtrans %}</td>
         <td>
-            <ul class="participant-list">
-                {%- for participant in participants -%}
-                    <li>{{ participant.full_name }}</li>
-                {%- endfor -%}
-            </ul>
+            <div class="participant-list-wrapper">
+                <ul class="participant-list">
+                    {%- for participant in participants -%}
+                        <li>{{ participant.full_name }}</li>
+                    {%- endfor -%}
+                </ul>
+                <div class="gradient-layer"></div>
+                <div class="height-control"></div>
```

Comment:
Do we really need it in here? Have you tried creating the element on the fly? Not sure if it needs to be in the DOM to get a proper height. In any case, if it needs to be there I'd still create it with JS, append it, and delete it after getting the height.